### PR TITLE
changed verify_checksums_file string

### LINKED ISSUE #891: Improve clarity on the number of release signing keys
On the page `2017-01-01-download.md` (https://bitcoincore.org/en/download/) the following phrase suggests that there is a single release signing key,

`verify_checksums_file: "Verify that the checksums file is PGP signed by the release signing key:"`

but the surrounding lines indicate that there are multiple signing keys. I suggest it be changed to

`verify_checksums_file: "Verify that the checksums file is PGP signed by trusted keys:"`

to avoid confusion.

### PR DESCRIPTION
**Pull request by superstreber3**
Improved the clarity on the number of release signing keys based on Issue Closes #891

### CODE REVIEW

**Comment by mikeobank** [comment]
Maybe: _Verify that the checksums file is PGP signed by at least one trusted release signing key:_?

**Comment by fanquake** [comment]
@stickies-v

File: _posts/en/pages/2017-01-01-download.md [hunk 1]
```diff
@@ -103,7 +103,7 @@ choosing_builders: >
 
 release_key_obtained: "The output of the command above should say that one key was imported, updated, has new signatures, or remained unchanged."
 
-verify_checksums_file: "Verify that the checksums file is PGP signed by the release signing key:"
+verify_checksums_file: "Verify that the checksums file is PGP signed by a trusted release signing key:"
```
Comment (stickies-v) [inline _posts/en/pages/2017-01-01-download.md:106]: This implies that there is such a thing as a trusted key in the absolute sense. Suggested rephrase:

```suggestion
verify_checksums_file: "Verify that the checksums file is PGP signed by a sufficient amount of keys you trust and have imported into your keychain."
```

**Review by stickies-v — COMMENTED** [review]
Concept ACK, suggested to rephrase that trust is personal

**Comment by MarcoFalke** [comment]
Are you still working on this?

File: _posts/en/pages/2017-01-01-download.md (hunk 1 above)
Comment (superstreber3) [reply _posts/en/pages/2017-01-01-download.md]: ACK

**Review by MarcoFalke — APPROVED** [review]
lgtm

**Comment by superstreber3** [comment]
Sorry, it was by mistake

**Comment by MarcoFalke** [comment]
cc @fanquake Can this be merged or does it need squash? (If yes, please squash your commits according to https://github.com/bitcoin/bitcoin/blob/master/CONTRIBUTING.md#squashing-commits)

**Comment by fanquake** [comment]
Thanks. I've cherry-picked & squashed your change, with an updated commit message, in #946.

**Pull request closed without merging**